app/pods/secure/products/product/template.hbs

```diff
@@ -38,69 +38,48 @@
         cols="80"
         rows="3"
       />
-      <!--  -->
     </Fieldset>
 
     <hr>
 
     <Fieldset>
-      <!-- <Label
-        @text="Can be ordered?"
-      /> -->
-      <Input::Boolean
-        @value={{product.orderable}}
-        @textTrue="Can be ordered"
-        @textFalse="Out of production"
-        @onToggle={{fn (mut product.orderable)}}
-      />
-    </Fieldset>
-
-    <Fieldset>
-      <!-- <Label
-        @text="Public?"
-        @hint="If yes, this product will be shown on app.interflux.com. If no, this product is hidden from the world and can only be seen by Interflux members who are logged in on admin.interflux.com."
-      /> -->
-      <Input::Boolean
+      <Input::Checkbox
         @value={{product.public}}
-        @textTrue="Public"
-        @textFalse="Private"
+        @textLabel="Visible on website"
         @onToggle={{fn (mut product.public)}}
       />
     </Fieldset>
 
     <Fieldset>
-      <!-- <Label
-        @text="Featured?"
-      /> -->
-      <Input::Boolean
-        @value={{product.featured}}
-        @textTrue="Featured"
-        @textFalse="Normal"
-        @onToggle={{fn (mut product.featured)}}
+      <Input::Checkbox
```

Comment:
I'm on the fence whether to name these components after the UI (checkbox, toggle, ...) or after the data type (boolean, string, one-to-many). Booleans could be set with toggles, checkboxes and buttons alike. In this case `Input::Boolean` is already taken by a toggle UI, so you did the right thing namespacing to `Input::Checkbox`.

Reply:
It was my assumption that similar to Salesforce's ootb components there would be a 'CheckboxGroup' or similar to support groups of 'real' checkbox inputs